File: trunk/PolarMathWinRT/Evaluation/Spectrum/ULFPercents.cs
using HrmMath.Data;
using System;

namespace HrmMath.Evaluation.Spectrum
{
    public sealed class ULFPercents : IEvaluation
    {
        public Index Name
        {
            get { return Index.ULFPercents; }
            set { }
        }

        public object Evaluate(SessionData training)
        {
            var ulf = Convert.ToDouble(training.Evaluate( new ULF() ));
            var tp = Convert.ToDouble(training.Evaluate( new TP() ));
            return (ulf / tp) * 100;
        }
    }
}

File: PolarMath/src/org/macondo/polar/evaluation/spectrum/ULFPercents.java
package org.macondo.polar.evaluation.spectrum;

import org.macondo.polar.data.Training;
import org.macondo.polar.evaluation.Evaluation;

public class ULFPercents implements Evaluation<Double> {
	
	public Double evaluate(Training training) {
		double ulf = training.evaluate(new ULF());
		double tp = training.evaluate(new TP());
		return (ulf / tp) * 100;
	}

}
